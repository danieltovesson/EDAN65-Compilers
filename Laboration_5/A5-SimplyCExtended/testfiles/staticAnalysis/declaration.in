int main() {
  int a;
  int b = 4*5;
}
