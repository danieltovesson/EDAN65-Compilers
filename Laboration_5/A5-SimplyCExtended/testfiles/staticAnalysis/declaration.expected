Declaration: a = 0
Assignment: b = 20
